src/test/resources/com/puppycrawl/tools/checkstyle/checks/indentation/InputCustomAnnotation.java

```diff
@@ -0,0 +1,83 @@
+package com.puppycrawl.tools.checkstyle.checks.indentation; //indent:0 exp:0
+
+import java.lang.annotation.ElementType; //indent:0 exp:0
+import java.lang.annotation.Repeatable; //indent:0 exp:0
+import java.lang.annotation.Retention; //indent:0 exp:0
+import java.lang.annotation.RetentionPolicy; //indent:0 exp:0
+import java.lang.annotation.Target; //indent:0 exp:0
+
+public @interface InputCustomAnnotation { //indent:0 exp:0
+} //indent:0 exp:0
+
+     class TestClass { //indent:5 exp:0 warn
+String author; //indent:0 exp:4 warn
+} //indent:0 exp:0
+
+     interface TestInterface {} //indent:5 exp:0 warn
+
+     enum TestEnumInput {} //indent:5 exp:0 warn
+
+     @interface AnnotationWithComments { //inside comments } //indent:5 exp:0 warn
+     } //indent:5 exp:0 warn
+     @interface AnnotationWithStringProperty { //indent:5 exp:0 warn
+String author(); //indent:0 exp:4 warn
+     } //indent:5 exp:0 warn
+
+   @Retention(RetentionPolicy.SOURCE) //indent:3 exp:0 warn
+@interface AnnotationWithDefaultStringAndAnnotation { //indent:0 exp:0
+String author() default "author"; //indent:0 exp:4 warn
+     } //indent:5 exp:0 warn
+
+     @interface //WRONG //indent:5 exp:5
+     AnnotationWithLineWrap { //indent:5 exp:0 warn
+    String author(); //indent:4 exp:4
+} //indent:0 exp:0
+
+     @Target(ElementType.METHOD) //indent:5 exp:0 warn
+   @Retention(RetentionPolicy.RUNTIME) //indent:3 exp:0 warn
+@interface AnnotationWithTarget { //indent:0 exp:0
+    String author() default "author"; //indent:4 exp:4
+     String book(); //indent:5 exp:4 warn
+   @Retention(RetentionPolicy.SOURCE) //indent:3 exp:4 warn
+@Repeatable(RepeatableInner.class) //indent:0 exp:4 warn
+@interface //indent:0 exp:0
+        AnnotationInnerLineWrap //WRONG exp 8 //indent:8 exp:7 warn
```

Comment:
We don't need these `WRONG` comments if there is a warning/violation on the line.